scripts/pylib/twister/twisterlib/statuses.py

```diff
@@ -52,3 +68,384 @@ def get_color(status: TwisterStatus) -> str:
     FAIL = 'failed'
     PASS = 'passed'
     SKIP = 'skipped'
+
+
+class TwisterStatusMachineInner(StateMachine):
+    '''
+        twister status statue machine
+    '''
+    none_statue = State(str(TwisterStatus.NONE), initial=True)
```

Comment:
statue? Shouldn't this be 'status'?

Reply:
should be state, this is the concept of pystatemachine, let me update this.